transmart-batch/src/main/groovy/org/transmartproject/batch/facts/DummyFactGenerator.groovy

```diff
@@ -72,4 +110,30 @@ class DummyFactGenerator extends AbstractItemCountingItemStreamItemReader<Clinic
 
     @Override
     protected void doClose() throws Exception {}
+
+    private List<Map<String, Object>> getAssayIds(String sample_id) {
+        List result = jdbcTemplate.queryForList """
```

Comment:
What about making the result of the query the main data source, instead of reading from `subjectConceptMappingFileRows`?